metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableConnectionBuilder.java

```diff
@@ -117,6 +124,7 @@ public BigtableConnection call() throws Exception {
 
     public String toString() {
         return "BigtableConnectionBuilder(project=" + this.project + ", instance=" + this.instance
-               + ", credentials=" + this.credentials + ")";
+            + ", profile=" + (this.profile != null ? this.profile : "null")
```

Comment:
I think this is unnecessary => "profile =" + this.profile  is sufficient.

Reply:
I wasn't sure if null pointer exception will kick in here if it is null.

Reply:
fixed